src/GitVersionTfsTask/GitVersion.ps1

```diff
@@ -54,6 +54,13 @@ if($additionalArguments)
   $argsGitVersion = $argsGitVersion + " $additionalArguments"
 }
 
+if(!$additionalArguments -or !$additionalArguments.Contains("-debug"))
+{
+  if([string]::IsNullOrEmpty($env:SYSTEM_DEBUG) -or $env:SYSTEM_DEBUG -ne "true"){
```

Comment:
I see where you were going with this now after looking through the commits. I am going to remove for now. I think we should extend the VSTS task to expose the log level.